pkg/bufferdecoder/protocol.go

```diff
@@ -110,12 +110,3 @@ type SlimCred struct {
 func (s SlimCred) GetSizeBytes() uint32 {
 	return 80
 }
-
-type HookedSyscallData struct {
-	SyscallName string
-	ModuleOwner string
-}
-type SymbolData struct {
-	SymbolName    string
-	SymbolAddress string
-}
```

Comment:
Aren't these part of the protocol from the bpf code to userspace anymore?

Reply:
Hmmm - we use `trace.HookedSymbolData` instead. bpf code sends unsigned long pointers that are casted in the usermode to `trace.HookedSymbolData`.